Cookbook/Technical-Documents/OutOffOfficeRequest.md

```diff
@@ -0,0 +1,60 @@
+# Requesting and Notifying Out Of Office time
+
+Someone who is Out Of Office means that is not in their usual work place at their usual working hours. This can be when someone goes on holidays or when someone that usually works in the office is working from somewhere else.
+
+OOO time has to be requested and approved by your Line Manager and your Project Manager.
+
+There are different types of OOO and each one of them have different requirements for requesting and for being notified.
+
+## Holidays 
+
+As per contract, we are entitled to 25 holiday days per year. You can book your holidays at your convience, you might just need to sync first with your Squad and the other iOS engineers to see if everybody is OK with it. 
+
+Ideally, you should give at least 2 weeks notice before taking any holiday but there are exceptions for emergency situations. If you want to book more than 5 consecutive days you ideally should give 4 weeks notice.
+
+### Steps to request
+
+1. Ask for approval from your PM - only if approved you can proceed to the next steps.
+2. Communicate it to your line manager and mention your PM approval
+3. Request in Bamboo
+
+### Steps to notify
+
+1. [Send an event](#steps-to-configure-the-outlook-event) to the iOS Team Calendar and to your Squad email group
+1. Update the [Team Plan document](https://docs.google.com/spreadsheets/d/1kdY3edy_TeqIGH_7VnZzElxgVo_qD2z4EF-arWNShyw/edit?usp=sharing)
+1. Notify the iOS Team and your Squad via Slack on the working day before you go on holidays
+
+If there is a big impact in the iOS Team or in your Squad due to you being away, please communicate a few days before in the team specific slack channel and do a handover if needed.
+
+#### Steps to Configure the Outlook event
+1. Open Outlook 
+1. Select Calendar
+1. Select Appointment
+1. Select Invite
+1. To: `<iOS calendar email group>`, `<your squad email group>`
+1. Subject: `<Your name>` Holidays 🌴
+1. Duration: All-day event
+1. Show as: Free
```

Comment:
This has the benefit of people accidentally accepting your invite not being marked as Busy (as outlook would consider that accepting a meeting invite would make you part of it so busy at the time of appointment), but has the drawback of not marking the person actually being on holidays as Busy.

This means that other people wanting to later organise any meetings while that person is away won't see that person as "not available" in the Scheduling view.

Two possible suggestions to fix that:

1. Either keep that invite to communicate with the team as "Free" but then we need to also create an additional all-day Appointment event in our own calendar only, without inviting anybody, marking us as Busy. This would make us have two similar events in our calendars (one for the invite shared with the group as free, one only local marking us as busy) but would avoid other people accidentally "accepting" our shared "meeting" invite to be marked as busy themselves
2. Or we should select "Busy" and not "Free" in that invite we send to the team, but need to make sure to communicate that the team receiving that invite should NOT accept NOR reject the invite, but just delete the email. Deleting the email will neither accept the invite (preventing you to be marked as busy) nor reject it (preventing it from being removed from the calendar), it will just keep the event in the calendar without marking you busy. The benefit of this solution is having only one event, the drawback being that it's easy to accept by mistake / force of habit and be marked as busy.

Reply:
I can see that having a person busy in the calendar can be useful, although I don't feel that the options presented are feasible:
1. We would have to create 2 events and as you mentioned:
>This would make us have two similar events in our calendars (one for the invite shared with the group as free, one only local marking us as busy)
2. I feel this can be very prone to error and I'm afraid people start to accept/reject meetings by mistake.

So at the moment, if anyone invites to a meeting and one of the attendees is going to be on holiday, the attendee should reject the meeting and should mention why, including the days this person is on holidays. 
In the event that the invite is sent when the person is already on holidays, then maybe we can add in our process for everyone to add an automatic reply to their outlook stating the days that he/she is on holiday. This message would be sent every time someone tries to invite that person to a meeting. This would also be useful, not for meeting invites, but also for any other email sent to that person.

Reply:
I agree that adding a point about setting up email auto-reply is a good idea.

Reply:
Great! I will update this document with that!

Reply:
Done 🙂